src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -383,6 +389,14 @@ public boolean equals(Object obj) {
             return false;
         }
 
+        if (questionDescription == null) {
+            if (other.questionDescription != null) {
```

Comment:
These can be combined into `if (questionDescription == null || other.questionDescription != null) {`.